src/test/java/resources/batch.feature

```diff
@@ -14,6 +14,14 @@ Feature: Batch Details API test
     Then I should get non empty response
     And I should have "PollingPath" and "SuggestedCallbackSeconds" in response
 
+  Scenario: Batch aggregate API Test
+    Given I have the demo csv file "test.csv"
+    And I have tenant as "gorilla"
+    When I call the batch transactions endpoint with expected status of 200
+    Then I should get non empty response
+    And I fetch batchId from response
+    When I call the batch aggregate API with expected status of 200
+    Then I should get non empty response
```

Comment:
you should open and check the response

Reply:
@arkadasfynarfin : Please address this review comment.